src/evergreen/api.py

```diff
@@ -5,6 +5,11 @@
 import logging
 import time
 
+try:
+    from json.decoder import JSONDecodeError
+except ImportError:
```

Comment:
Might be helpful to include a comment like here https://github.com/evergreen-ci/evergreen.py/pull/21/files#diff-605ef5f63c2af7e5816682aaac567219R16

Reply:
done